control/tests/index.js

```diff
@@ -1,4 +1,4 @@
-authManager.enforceLogin();
+/*authManager.enforceLogin();
```

Comment:
if we are not using auth manager through the app lets remove this file as it seems redundant